daemon/process/CMakeLists.txt

```diff
@@ -126,20 +126,20 @@ endif()
 install(
         FILES       "${DOBBY_SETTINGS_FILE}"
         RENAME      dobby.json
-        DESTINATION /etc/sky/ )
+        DESTINATION /etc/rdk/ )
 
 
 
 # Install a systemd launch service config and enables it by default
 install(
-        FILES       sky-dobby.service
+        FILES       rdk-dobby.service
```

Comment:
As already pointed out by @TeknoVenus, `dobby.service` is preferred over `rdk-dobby.service`. The same can be found in quite a few other places as well.

As a note - this will require a change to the dobby recipe as well.